tilelang/autodd.py

```
@@ -0,0 +1,1180 @@
+
+    @override
+    def task_generator(self) -> Iterable[Task]:
+        for task in self.generator():
+            lines = [line for line in self.source.splitlines() if line.strip() != ""]
+            new_lines = [line for idx, line in enumerate(lines) if idx not in task.applied]
+            source = "\n".join(new_lines)
+            try:
+                ast.parse(source)
+            except Exception as _:
+                # self.update(task, is_interesting=False)
+                continue
```

Ruff (0.14.10)

`try`-`except`-`continue` detected, consider logging the exception

(S112)